Mathlib/SetTheory/ZFC/Ordinal.lean

```diff
@@ -12,27 +13,24 @@ import Mathlib.SetTheory.ZFC.Basic
 This file works towards the development of von Neumann ordinals, i.e. transitive sets, well-ordered
 under `∈`.
 
-We currently only have an initial development of transitive sets and ordinals. Further development
-can be found on the Mathlib 3 branch `von_neumann_v2`.
-
 ## Definitions
 
 - `ZFSet.IsTransitive` means that every element of a set is a subset.
 - `ZFSet.IsOrdinal` means that the set is transitive and well-ordered under `∈`.
 
-## TODO
+## Todo
```

Comment:
I think TODO is the more common spelling in mathlib. To-do is also good to me.

Reply:
Oh alright, I'll revert.